_pages/en_US/troubleshooting.txt

```diff
@@ -96,18 +102,31 @@ Browser based exploits (such as browserhax or 2xrsa) are often unstable and cras
 
 ## Black screen on SysNAND boot
 
+### What you need
+
+* [`remove_extdata.gm9`]({{ "/gm9_scripts/remove_extdata.gm9" | absolute_url }})
+
+### Instructions
+
+Note that if you have any payload files other than `GodMode9.firm` in the `/luma/payloads/` folder on your SD card, holding (Start) on boot will display a "chainloader menu" where you will have to use the D-Pad and the (A) button to select "GodMode9" for these instructions. 
+{: .notice--info}
+
 1. Try booting with your SD card out, and then reinserting it after booting
     1. Power off your device
     1. Remove your SD card from your device
     1. Power on your device
     1. When the home menu appears, reinsert your SD card into your device
-    1. If this worked, you should clear Home Menu's extdata by navigating to the following folder on your SD card: `/Nintendo 3DS/(32 Character ID)/(32 Character ID)/extdata/00000000/`
-        + EUR Region: Delete `00000098`
-        + JPN Region: Delete `00000082`
-        + USA Region: Delete `0000008f`
-        + CHN Region: Delete `000000A1`
-        + KOR Region: Delete `000000A9`
-        + TWN Region: Delete `000000B1`
+    1. If this worked, you should clear Home Menu's extdata by following these instructions:
+        1. Power off your device
+        1. Launch GodMode9 by holding (Start) during boot
+        1. Press (Home) to bring up the action menu
+        1. Select "Scripts..."
+        1. Select "remove_extdata"
+        1. When prompted, press (A) to proceed
+        1. Press (A) to unlock SysNAND (lvl1) writing, then input the key combo given
```

Comment:
The user will not be prompted to unlock SysNAND writing because this is just deleting something from the SD card.